packages/utils/src/signatures.ts

```diff
@@ -106,3 +109,35 @@ export async function isValidEip1271Signature(
 function generateJsonRpcId() {
   return Date.now() + Math.floor(Math.random() * 1000);
 }
+
+export function extractSolanaTransactionId(solanaTransaction: string): string {
+  const binary = atob(solanaTransaction);
```

Comment:
what's `atob`?

in Kotlin i'm trying to decode base64 if fails base58

Reply:
`atob` decodes base64 string. I added a bit extra validation just in case to avoid handling dummy data. 
For example, atob throws if the string is not valid base64 encoding, and the validation after that will filer base64 encoded non transaction values

Reply:
got it
phantom docs says that these signed transactions can be base58 encoded but our solana rpc docs say that they can be base64
i think we should add decoding for both